addons/mail/static/src/models/rtc_controller.js

```diff
@@ -80,6 +80,17 @@ registerModel({
         _computeIsSmall() {
             return Boolean(this.callViewer && this.callViewer.threadView.compact && !this.callViewer.isFullScreen);
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeMicrophoneTitle() {
+            if (this.messaging.rtc.currentRtcSession.isMute) {
```

Comment:
You should ensure that `currentRtcSession` is defined before trying to read `isMute`.

Reply:
Same as in the `headphoneTitle`, it wasn't guarded in template, so I assume it never happens to have `RtcController` without a `currentRtcSession`